addon/components/frost-select-dropdown.js

```diff
@@ -401,6 +442,27 @@ export default Component.extend({
       }
     })
 
+    Array.from(secondaryTextElements).forEach((secondaryTextElement, index) => {
```

Comment:
This part of the code looks a lot like line 382. Is there a way to avoid duplication?